File: python_Exercise_problems/preschool_kid.py
pebbles=[1,2,3,4,4,3,2,1]
# pebbles=[1,2,3,4,4,4,3,2,1,2]


list=[]
counts=0

for item in pebbles:
    counts=pebbles.count(item)
    list.append(counts)


key=list[0]
for i in list:
    if(i!=key):
        print("false")
        exit()

print("true")

File: python_Exercise_problems/preschool_kid.c
#include<stdio.h>
#include<stdlib.h>

int main(){
    int pebbles[]={1,1,2,2};
    // int pebbles[]={1,2,3,4,4,4,3,2,1,2};
   
    int count=0;
    int counts[8];
    for(int i=0;i<8;i++){
        int temp=pebbles[i];
        
        for(int j=0;j<8;j++){
            if(temp==pebbles[j]){
                count++;
            }
        }
           counts[i]=count;
           count=0;
    }
    int k=counts[0];
    int flag=1;
    
    for(int i=0;i<8;i++){
        if(counts[i]!=k){
            flag=0;
            break;
        }
    }
    if(flag==1){
        printf("true");
    }else{
        printf("false");
    }
    return 0;
}